src/eckit/exception/Exceptions.h

```diff
@@ -285,40 +263,43 @@ inline T SysCall(T code, const char* msg, const char* file, int line, const char
     return code;
 }
 
-
 template <class T>
 inline long long SysCall(long long code, const char* msg, const T& ctx, const char* file, int line, const char* func) {
     if (code < 0) {
         std::ostringstream os;
-        os << ctx;
-        throw FailedSystemCall(os.str(), msg, CodeLocation(file, line, func), errno);
+        os << msg << " [" << ctx << "]";
+        throw FailedSystemCall(os.str(), CodeLocation(file, line, func), errno);
     }
     return code;
 }
 
-
 inline void ThrCall(int code, const char* msg, const char* file, int line, const char* func) {
-    if (code != 0) {  // Threads return errno in return code
+    if (code != 0) {
+        // Threads return errno in return code
         handle_panic(msg, CodeLocation(file, line, func));
     }
 }
 
-/// This functions hides that assertions may be handled by a throw of AssertionFailed
-/// which is incompatible with using ASSERT inside destructors that are marked as no-throw,
-/// which is the default behavior in C++11.
-/// If env var ECKIT_ASSERT_ABORTS is set the assertion will abort instead of throwing.
-/// In the future this will become the default behavior.
+/**
+ * This functions hides that assertions may be handled by a throw of AssertionFailed
+ * which is incompatible with using ASSERT inside destructors that are marked as no-throw,
+ * which is the default behavior in C++11.
+ * If env var ECKIT_ASSERT_ABORTS is set the assertion will abort instead of throwing.
+ * In the future this will become the default behavior.
+ */
```

Comment:
Is there a specific reason to change from this to the java-doc style? `///` Is also a accepted style in Doxygen

Reply:
Ah well this is just personal taste. Yes either syntax is accepted, but /** */ describes a block so it reads as a unit, while /// /// reads as multiple lines. It really is a detail, if you feel strognly on this I can change it back.

Reply:
No, I was just curious if there is any specific reason, because we agreed to use `///` in FDB end of 2024.